frontend/src/components/curation/ObjectContext.test.tsx

```
@@ -139,6 +139,58 @@ describe('ObjectContext', () => {
+
+    expect(screen.getByRole('link', { name: 'Mbzt' })).toBeInTheDocument();
+    expect(screen.getByRole('link', { name: 'CC BY-SA 4.0' })).toBeInTheDocument();
+  });
+
+  it('takes the credit away with a picture that did not arrive', () => {
+    // Not an edge case on this screen: most of what the queue holds is UNESCO,
+    // whose URLs largely answer 403 (#557), and the proxy surfaces that as an
+    // image error. A photographer named under a broken frame is the one claim
+    // the credit feature exists to avoid making.
+    render(<ObjectContext item={item({
+      image_url: 'http://commons.wikimedia.org/wiki/Special:FilePath/Mesha%20stele.jpg',
```

ESLint

[error] Using http protocol is insecure. Use https instead.

(sonarjs/no-clear-text-protocols)